app/_src/kubernetes-ingress-controller/guides/using-gateway-api.md

```diff
@@ -241,6 +241,16 @@ HTTPRoute resources are similar to Ingress resources: they contain a set of
 matching criteria for HTTP requests and upstream Services to route those
 requests to.
 
+https://github.com/Kong/kubernetes-ingress-controller/issues/3314
+
+{:.important}
+> The Gateway API specification binds HTTPRoutes to one or more listeners in
+> a Gateway resource. These listeners have a specific port, and HTTPRoutes
+> should only be available on their listeners' ports per the specification.
+> Kong's HTTP proxy implementation [does not support this](https://github.com/Kong/kubernetes-ingress-controller/issues/3314):
+> if you configure multiple HTTP ports, Kong serves all HTTP routes on all
+> ports.
```

Comment:
```suggestion
{% if_version gte:2.9.x %}
{:.important}
> The Gateway API specification binds HTTPRoutes to one or more listeners in
> a Gateway resource. These listeners have a specific port, and HTTPRoutes
> should only be available on their listeners' ports per the specification.
> Kong's HTTP proxy implementation [does not support this](https://github.com/Kong/kubernetes-ingress-controller/issues/3314):
> if you configure multiple HTTP ports, Kong serves all HTTP routes on all
> ports.
{% endif_version %}
```
If this note is specific to 2.9.x or later, we can make this conditional so it only displays for that version or later.

Reply:
This is true for all existing versions. We just didn't call it out before.